libs/sql-schema-describer/src/walkers.rs

```diff
@@ -153,6 +153,15 @@ impl<'a> TableWalker<'a> {
         self.table().foreign_keys.len()
     }
 
+    /// Traverse to an index by index.
+    pub fn index_at(&self, index_index: usize) -> IndexWalker<'a> {
```

Comment:
Taking suggestions for alternative naming here and in general.

Reply:
:yodawg: